styleguide/source/assets/scss/02-molecules/_promo.scss

```diff
@@ -16,6 +16,23 @@
     border: solid 1px $gray-50;
   }
 
+  &--inline {
+    border-top: solid 1px $gray-50;
+    border-bottom: solid 1px $gray-50;
+    padding-left: 0;
+    @extend .col-xs-12;
+    @extend .col-sm-4;
+    @include gutter($padding-top-full...);
+    @include gutter($padding-right-full...);
+    @include gutter($padding-bottom-full...);
+    @include gutter($margin-right-full...);
+    @include gutter($margin-bottom-full...);
```

Comment:
As a best practice you should be putting the `@extend` and `@include` rules first and then override them with your own rules for a thing as part of the cascade.